src/main/java/org/openstreetmap/atlas/utilities/filters/AtlasEntityPolygonsFilter.java

```diff
@@ -0,0 +1,168 @@
+package org.openstreetmap.atlas.utilities.filters;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.openstreetmap.atlas.geography.Polygon;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasEntity;
+import org.openstreetmap.atlas.geography.converters.PolygonStringConverter;
+import org.openstreetmap.atlas.geography.converters.WkbPolygonConverter;
+import org.openstreetmap.atlas.geography.converters.WktPolygonConverter;
+import org.openstreetmap.atlas.geography.index.QuadTree;
+import org.openstreetmap.atlas.streaming.readers.GeoJsonReader;
+import org.openstreetmap.atlas.streaming.readers.json.serializers.PropertiesLocated;
+import org.openstreetmap.atlas.streaming.resource.StringResource;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+import org.openstreetmap.atlas.utilities.conversion.StringConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.io.WKBReader;
```

Comment:
It looks like you are using the WKB converter below... I can't see where this import is used?

Reply:
WKB string are displayed in Hex format with two bytes per string character. This converts the Hex string to bytes for the converter [here](https://github.com/osmlab/atlas/pull/85/files#diff-327065d3a2dcb65a8d798a639325a5d9R86)